tools/detect_nonexistent_testdata

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+FILES="${@}"
+success=1
+for FILE in $FILES; do
+	if  ! [ -f "$FILE" ]; then
+		echo "Error! $FILE does not exist."
+		success=0
+	elif ! [ -s "$FILE" ]; then
+		echo "Warning! The test_data file $FILE is empty."
```

Comment:
Why this is a warning?

Reply:
I wasn't sure if I should make it an error so that the check would fail. You think it should fail if test data file is empty?

Reply:
i was just curious and i am asking for clarity. Ignore this.